src/coreclr/vm/comcallablewrapper.h

```diff
@@ -499,6 +499,7 @@ struct ComMethodTable
     // Accessor for the IDispatch information.
     DispatchInfo* GetDispatchInfo();
 
+#ifndef DACCESS_COMPILE
```

Comment:
assume this is unrelated to w^x change?

Reply:
They are related. They exclude methods that are not used by DAC and that would otherwise unnecessarily pull in the ExecutableWriterHolder into DAC.